edb/edgeql/compiler/cast.py

```diff
@@ -113,8 +114,10 @@ def compile_cast(
             ir_set, orig_stype, new_stype, srcctx=srcctx, ctx=ctx)
 
     else:
-        json_t = ctx.env.get_track_schema_object('std::json')
-
+        json_t = cast(
+            s_objects.InheritingObjectBase,
```

Comment:
The actual type is going to be `edb.schema.scalars.ScalarType`

Reply:
Yeah, we can change this but we only need `issubclass` below. Since it's a cast to get Mypy quiet, doesn't really matter much.